bot/handlers/commands.py

```diff
@@ -53,7 +57,16 @@ async def cmd_read_ruled(message: Message):
 
 @router.message(Command(commands=['stats']))
 async def cmd_stats(message: Message):
-    await message.answer('# todo statistics menu')
+    start, end = ps.get_date_period_from_message(message.text)
+    if start > end:
+        start, end = end, start
+    text = f"Статистика за період з <b>{start.date()}</b> по <b>{end.date()}</b>"
+    if start == end:
+        text = f"Статистика за <b>{start.date()}</b>"
```

Comment:
I'm not sure but it looks like in case `start == end` text would be redefined. So does it make sense to initialize `text` before?